app/vmui/packages/vmui/src/components/Home/Configurator/Query/QueryEditor.tsx

```diff
@@ -86,24 +87,33 @@ const QueryEditor: FC<QueryEditorProps> = ({
     if (target?.scrollIntoView) target.scrollIntoView({block: "center"});
   }, [focusOption]);
 
-  return <Box ref={autocompleteAnchorEl}>
+  return <Box ref={autocompleteAnchorEl} sx={{position: "relative", zIndex: focusField ? 2 : 1}}>
```

Comment:
It looks like this leads to the following bug:

![vmui_bug](https://user-images.githubusercontent.com/283442/154155726-3b0f481b-3cdc-47de-bb79-bd03757c5faa.png)

Reply:
Fixed